eskip/string.go

```diff
@@ -78,9 +80,17 @@ func argsString(args []interface{}) string {
 	return strings.Join(sargs, ", ")
 }
 
+func tlsIsEmpty(t tls.Certificate) bool {
+	return reflect.DeepEqual(t, tls.Certificate{})
+}
+
 func (r *Route) predicateString() string {
 	var predicates []string
 
+	if !tlsIsEmpty(r.TLSCert) {
+		predicates = appendFmtEscape(predicates, `TLS(True)`, `"`)
```

Comment:
Why do we need this? Also I do not see new `TLS` predicate being introduced.

Reply:
Resolving this since not using routes to determine TLS certificates